### PR TITLE
Prettify ls

### PR DESCRIPTION
**Pull request by syedbabart**
Files and directories can be displayed in lines of 3 each.

### CODE REVIEW

**Comment by EeshaArif** [comment]
image
Take a look at this please

File: commands/ls.py [hunk 1]
```diff
@@ -8,9 +8,29 @@ class LSCommand(BaseCommand):
     command = 'ls'
 
     def run(self):
-        self.log(DirectoryDao.get_directories_from_current_directory(
-            self.context.current_directory), prefix=False
-        )
-        # logging files - temp
-        self.log(FileDao.get_files_from_current_directory(
-            self.context.current_directory), prefix=False)
+        def display_lists(dir_list):
```
Comment (EeshaArif) [inline commands/ls.py:11]: The function can created outside `run()` and called by `self.display_lists()`

**Review by EeshaArif — CHANGES_REQUESTED** [review]
~

**Comment by syedbabart** [comment]
Pre-commit checks have been successful. Please review now.

**Review by EeshaArif — APPROVED** [review]
~

**Pull request merged**